acid/features/history/controller.py

```diff
@@ -14,10 +14,10 @@
 @builds.route('/builds/<int:page>')
 @db_session
 def show_builds_history(page=1):
-    per_page = current_app.config['buildset']['per_page']
+    per_page = current_app.config['history']['pagination']['per_page']
```

Comment:
Maybe put:
config = current_app.config
And later refer just to config, like in other change in controller.py